app/Http/Controllers/API/PlanController.php

```diff
@@ -28,7 +28,10 @@ public function __construct(PlanService $service)
      */
     public function index(): JsonResponse
     {
-        return response()->json(['data' => PlanResource::collection($this->service->index()), 'success' => true, 'type' => 'success']);
+        return response()->json([
+            'data' => PlanResource::collection($this->service->index()),
+            'success' => true,
+            'type' => 'success']);
```

Comment:
![image](https://github.com/Kar9988/project-no-25/assets/41668915/05c31b5f-7926-4e0b-bf6c-1553746f66f0)
Like this